src/NzbDrone.Core.Test/DecisionEngineTests/AcceptableSizeSpecificationFixture.cs

```diff
@@ -171,50 +167,25 @@ public void should_return_true_if_unlimited_30_minute()
         [Test]
         public void should_return_true_if_unlimited_60_minute()
         {
-            GivenLastEpisode();
+            GivenLastAlbum();
 
-            series.Runtime = 60;
-            parseResultSingle.Series = series;
+            parseResultSingle.Artist = artist;
             parseResultSingle.Release.Size = 36857280000;
             qualityType.MaxSize = null;
 
             Subject.IsSatisfiedBy(parseResultSingle, null).Accepted.Should().BeTrue();
         }
 
         [Test]
-        public void should_treat_daily_series_as_single_episode()
-        {
-            GivenLastEpisode();
-
-            series.Runtime = 60;
-            parseResultSingle.Series = series;
-            parseResultSingle.Series.SeriesType = SeriesTypes.Daily;
-            parseResultSingle.Release.Size = 300.Megabytes();
-
-            qualityType.MaxSize = 10;
-
-            Subject.IsSatisfiedBy(parseResultSingle, null).Accepted.Should().BeTrue();
-        }
-
-        [Test]
         public void should_return_true_if_RAWHD()
```

Comment:
I think this is checking for an actual full disc rip image (ISO, etc). I wasn't sure if we would run across that in music scene